src/content/docs/style-guide/processes-procedures/delete-document.mdx

```diff
@@ -10,12 +9,12 @@ redirects:
 ---
 
 <Callout variant="caution">
-  When you delete a document, its content and all redirects to it are completely and irreversibly deleted. If you believe the document may be of use in the future, OR if you do not want to re-create its redirects, consider another method to [hide the document](/docs/new-relic-only/advanced-style-guide/processes-procedures/hiding-content) from public view.
+  When you delete a document, its content and its redirects will still be available in the GitHub commit history. In general, if you're not on the New Relic Docs team, please don't delete any docs. Instead, [file an issue](https://github.com/newrelic/docs-website/issues/new/choose) and someone on our team will help you out.
 </Callout>
 
 If you are certain you want to delete a document, do this:
 
-1. Locate the doc you want to delete. 
+1. Locate the MDX file of the doc you want to delete. 
 2. Save the doc's content in a file and attach it to an associated Jira (in case we need the doc again).
-3. PENDING: INFO ON REDIRECTS
+3. Find the most relevant doc and [add the deleted doc's URL as a redirect](/docs/style-guide/rename-or-redirect-document).
```

Comment:
Is this missing a step to delete it from the `.yml` files?

Reply:
Yup! Great catch, @bradleycamacho! I'll get that added.